interface/backend/api/views.py

```diff
@@ -27,3 +28,14 @@ class NoduleViewSet(viewsets.ModelViewSet):
 class ImageSeriesViewSet(viewsets.ModelViewSet):
     queryset = ImageSeries.objects.all()
     serializer_class = serializers.ImageSeriesSerializer
+
+
+class ImageSrcApiView(APIView): 
+    """ 
+    View list of images from dataset directory 
+    """ 
+    def get(self, request): 
+        """ 
+        Return a list of files and folders in dataset
+        """ 
+        pass
```

Comment:
Are you planning to implement or leave this stubbed out? Stub is acceptable if that's the plan, we'll just create another issue for actually implement this.

Reply:
We can create a new issue for this.